conftest.py

```diff
@@ -2,6 +2,14 @@
 from enum import Enum
 from supplier import models
 from client.models import Client
+from supplier_product.models import SupplierProduct
+from product.tests import test_product
+
+
+prod = test_product.product0
```

Comment:
You should instead move that fixture here. if it becomes too much for one PR, you can copy the fixture here and then create a small PR which deletes it from `test_product.py`